andraszakupszki-london.md

```diff
@@ -0,0 +1,9 @@
+# London, UK
+
+## Places to Eat
+- Borough Market
```

Comment:
Mainly healthy/organic street food. If you're in London I strongly recommend it.